staging/src/k8s.io/kubectl/pkg/drain/filters.go

```diff
@@ -133,8 +134,11 @@ func makePodDeleteStatusWithError(message string) podDeleteStatus {
 	}
 }
 
+// The filters are applied in a specific order, only the last filter's
+// message will be retained if there are any warnings.
 func (d *Helper) makeFilters() []podFilter {
 	return []podFilter{
+		d.skipDeletedFilter,
```

Comment:
Previously I had this filter appended to the end of the list, but that broke the tests here:
./staging/src/k8s.io/kubectl/pkg/cmd/drain/

The effect of this was very non-obvious, so I added the comment above.